README.md

```diff
@@ -19,12 +19,24 @@ is not present in the Fedora repository or `stage-digital-object-set` cannot be
 the program terminates with an error.
 
 For each digital object the last version of each (managed) datastream, a `fo.xml` and `cfg.json` file are downloaded.
-The `fo.xml` file includes the inline datastreams DC, EMD, AMD, PRSQL, DMD.
-- the datastream AUDIT is skipped completely
-- RELS-EXT is exported into to the "relations"-map in the file `cfg.json` ([Digital Object Configuration])
+The `fo.xml` file includes the inline datastreams `DC`, `EMD`, `AMD`, `PRSQL`, `DMD` as far as they are present. As
+for the other datastreams:
+
+* The datastream `AUDIT` is skipped completely.
+* `RELS-EXT` is exported into to the "relations"-map in the file `cfg.json` ([Digital Object Configuration])
   Fedora PIDs that reference digital objects in the same dataset are replaced by the appropriate SDO-name.
-- EASY-FILE-METADATA is downloaded separately as are eventual other inline datastreams not mentioned above
-- checksums and dataset related PIDs in the `fo.xml` are skipped
+* Other datastreams, such as `EASY_FILE_METADATA`,  are downloaded separately regardless whether they are inline or
+  managed.
+
+Checksums and dataset related PIDs in the `fo.xml` are skipped.
+
+**TODO (add to documentation)**:  
+    
+* **WHAT "dataset related PIDs" ARE REFERRED TO HERE?**
+* **What happens to the checksum of the inline datastreams exported in the foxml? (Exported along with the datastream?)**
+* **What happens to the checksum of datastreams downloaded separately? (For the sentence about the checksums above I
+  gather that these checksums are not downloaded. Are the checksum settings downloaded? I.e. will the checksum be
+  created when ingesting the staged dataset?)**
```

Comment:
To the point question: I remove `<foxml:contentDigest>` and after ingest I don't see it automagically regenerated!